posthog/hogql/context.py

```diff
@@ -16,12 +16,12 @@ class HogQLContext:
 
     # If set, will save string constants to this dict. Inlines strings into the query if None.
     values: Dict = field(default_factory=dict)
-    # List of field and property accesses found in the expression
-    field_access_logs: List[HogQLFieldAccess] = field(default_factory=list)
     # Did the last calls to translate_hogql since setting these to False contain any of the following
     found_aggregation: bool = False
     # Do we need to join the persons table or not
     using_person_on_events: bool = True
+    # swap person property access with "person_props" table in legacy hogql queries
+    part_of_legacy_query: bool = False
```

Comment:
probably I'm a bit out of context but "legacy hogql" sounds strange. is this to support things we do in the backend or previous functionality available but not used anywhere? saying this because I think we should be comfortable breaking things at this stage

Reply:
I'm still doubting here and already changed it to `legacy_person_property_handling` while you were reviewing. That's to just indicate that the prop handling is legacy (how "old" insights do it), not the queries themselves. I'm still not sure if that's the best name though. I'm not too happy with the `legacy`, as eventually everything is `legacy`, including this code, yet calling things "non-hogql-based-queries" is too verbose... 🤔